src/styles/globals.css

```diff
@@ -80,6 +80,15 @@
   svg {
     display: none;
   }
+  .scroll-container {
+    scroll-snap-type: y mandatory;
```

Comment:
There scroll-snap-type in tailwind, but me and Dhruv can't figure it out yet so for now this is fine, when we go over integration with other components we'll probably go over this.